rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -67,7 +67,7 @@ object Reduce {
       s: Sync[M],
       fTell: FunctorTell[M, Throwable],
       cost: _cost[M],
-      costWriter: _costWriter[M],
+      costLog: _costLog[M],
```

Comment:
From now on, before submitting a PR, please squash all the `rename` commits in a PR with the commit introducing the thing being renamed.

Reply:
Please fix this for this PR as well.

Reply:
Ok, will do